#!/usr/bin/env python
PACKAGE = "project_1"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

method_enum = gen.enum([ gen.const("Euler",   int_t, 0, "Euler integration method"),
                         gen.const("RK",      int_t, 1, "Runge-Kutta integration method")],
                         "Enum of integration methods")

gen.add("set_method", int_t, 0, "Integration method for odometry", 0, 0, 1, edit_method=method_enum)

exit(gen.generate(PACKAGE, "odometry", "parameters"))
